app/src/main/java/com/kodeco/android/countryinfo/ui/components/CountryDetailsScreen.kt

```diff
@@ -46,43 +42,50 @@ fun CountryDetailsScreen(navBackStackEntry: NavBackStackEntry, countries: List<C
             modifier = Modifier
                 .size(48.dp)
                 .offset(x = (-8).dp)
-                .clickable { navController.navigateUp() },
+                .clickable( onClick = onNavigateUp),
```

Comment:
super minor: add space

```suggestion
                .clickable( onClick = onNavigateUp ),
```

Sometimes linters will pick this up as a syntax warning.